tcclient/creds_test.go

```diff
@@ -12,7 +12,6 @@ func ExampleCredentials_CreateTemporaryCredentials() {
 	permaCreds := tcclient.Credentials{
 		ClientId:    os.Getenv("TASKCLUSTER_CLIENT_ID"),
 		AccessToken: os.Getenv("TASKCLUSTER_ACCESS_TOKEN"),
-		Certificate: "",
```

Comment:
This is a no-op change - setting empty string is same as not setting it, as it is a value type, it is the zero value.